surveys/2021-annual-survey/questions.md

```diff
@@ -932,22 +966,8 @@ Select all that apply:
 - < 10 year
 - > 10 years
 
-### Which operating systems do you use regularly for software development (not just Rust)?
+> Removed duplicate q's.
```

Comment:
FYI: there's no need to note this in the document itself. If you want you can leave these types of comments in the pull request on GitHub as comments (like I'm doing now).

Reply:
Thanks - My github newb-ness is more than showing through.